x-pack/platform/plugins/shared/streams_app/public/components/data_management/shared/condition_display.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { useEuiTheme, EuiFlexGroup, EuiFlexItem, EuiText, EuiBadge } from '@elastic/eui';
+import type { Condition, FilterCondition } from '@kbn/streamlang';
+import {
+  getFilterOperator,
+  getFilterValue,
+  isFilterConditionObject,
+  isAlwaysCondition,
+  isNeverCondition,
+  isAndCondition,
+  isOrCondition,
+  isNotCondition,
+  operatorToHumanReadableNameMap,
+} from '@kbn/streamlang';
+
+interface ConditionDisplayProps {
+  condition: Condition;
+  showKeyword?: boolean;
+  keyword?: string;
+}
+
+export const ConditionDisplay = ({
+  condition,
+  showKeyword = false,
+  keyword = 'WHERE',
+}: ConditionDisplayProps) => {
+  if (isAlwaysCondition(condition) || isNeverCondition(condition)) {
+    return null;
+  }
```

Comment:
I'm not sure about this. Shouldn't we display something for these cases to the user?